rust/datafusion/README.md

```diff
@@ -23,13 +23,44 @@ DataFusion is an in-memory query engine that uses Apache Arrow as the memory mod
 
 ## Usage
 
+
+#### Use as a lib
 Add this to your Cargo.toml:
 
 ```toml
 [dependencies]
 datafusion = "0.14.0-SNAPSHOT"
 ```
 
+#### Use as a bin
+##### Build your own bin(requires rust toolchains)
+```sh
+git clone https://github/apache/arrow
```

Comment:
I think you can remove the `git clone` step since this README is already in the cloned repo?

Reply:
Maybe a user will view README on Gtihub prior to clone this repo?
(At least I would prefer to read the README before clone something.)